.template-lintrc.js

```diff
@@ -1,6 +1,13 @@
 /* eslint-env node */
 'use strict';
 
+const OFF = false;
+
 module.exports = {
-  extends: 'recommended'
+  plugins: ['./node_modules/@smile-io/ember-styleguide/.template-lintrc'],
+  extends: ['@smile-io/ember-styleguide:recommended'],
+  rules: {
+    'attribute-indentation': OFF,
```

Comment:
I thought we'd want attribute indentation?

Reply:
ember-template-lint is broken for this rule. Once the fix lands, I'll be enabling this again.